dataservice/api/participant/resources.py

```diff
@@ -16,8 +16,15 @@ def get(self, kf_id):
         Gets a participant given a Kids First id
         """
         if kf_id is None:
+            def_size = current_app.config['DEFAULT_PAGE_SIZE']
```

Comment:
This bit of code will be common for all the resources. Can we move it somewhere in common so all resources can use it? Maybe you could even put this in a PaginationParams marshmallow schema class and do some validation on the pagination params like limit, offest (if we support it), etc.